packages/gatsby-source-npm-package-search/src/gatsby-node.js

```diff
@@ -0,0 +1,79 @@
+const algoliasearch = require(`algoliasearch`)
+const crypto = require(`crypto`)
+
+var client = algoliasearch(`OFCNCOG2CU`, `f54e21fa3a2a0160595bb058179bfb1e`)
```

Comment:
Use Const

Reply:
Get the values here from plugin options

Reply:
Or... maybe not actually — it'd be nice if this was just plug-n-play. Let's leave them hard-coded for now so people can just install the plugin and it works.